src/components/Text/DefaultText.tsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
 
-import { Text } from 'react-native'
+import { StyleProp, Text, TextProps, TextStyle } from 'react-native'
 
 import styles from './styles'
 
-interface Params {
-  text: string
-  additionalStyles: object
+export interface ITextProps extends TextProps {
+  children?: string
```

Comment:
TextProps already has children prop, why do override it?

Reply:
agree